specification/securityinsights/resource-manager/Microsoft.SecurityInsights/stable/2023-11-01/ContentTemplates.json

```diff
@@ -0,0 +1,353 @@
+          },
+          {
+            "$ref": "#/parameters/templateIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/2.0/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "templateList": {
+      "description": "List of all the template.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Array of templates.",
+          "items": {
+            "$ref": "#/definitions/templateModel"
+          },
+          "type": "array"
+        },
+        "nextLink": {
+          "description": "URL to fetch the next page of template.",
+          "readOnly": true,
+          "type": "string"
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "templateModel": {
+      "allOf": [
+        {
+          "$ref": "../../../common/2.0/types.json#/definitions/ResourceWithEtag"
+        }
+      ],
+      "description": "Template resource definition.",
+      "properties": {
+        "properties": {
+          "description": "template properties",
+          "$ref": "#/definitions/templateProperties",
+          "x-ms-client-flatten": true
+        }
+      },
+      "type": "object"
+    },
+    "templateProperties": {
+      "allOf": [
+        {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/templateBaseProperties"
+        },
+        {
+          "$ref": "#/definitions/templateAdditionalProperties"
+        }
+      ],
+      "description": "Template property bag.",
+      "required": [
+        "contentId",
+        "version",
+        "displayName",
+        "contentKind",
+        "source",
+        "packageId",
+        "packageVersion",
+        "contentProductId"
+      ],
+      "type": "object"
+    },
+    "templateAdditionalProperties": {
+      "description": "additional properties of product template.",
+      "type": "object",
+      "properties": {
+        "mainTemplate": {
+          "$ref": "#/definitions/mainTemplate",
+          "description": "The JSON of the ARM template to deploy active content. Expandable."
+        },
+        "dependantTemplates": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/templateProperties"
+          },
+          "description": "Dependant templates. Expandable.",
+          "readOnly": true,
+          "x-ms-identifiers": [
+            "contentId"
+          ]
+        }
+      }
+    },
+    "mainTemplate": {
+      "description": "The JSON of the ARM template to deploy active content",
+      "type": "object"
```

Comment:
Why is it a free-form JSON object here? Please define the schema so that customers know how to provide or parse this payload.

Reply:
Please check my previous comments.  Although it has a different name, but it is also an ARM templates to create a type of resources under securityInsights. https://github.com/Azure/azure-rest-api-specs/pull/26873#discussion_r1419800170